apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -93,13 +82,16 @@ public static function get_instance() {
 	/**
 	 * Determines whether provided theme supports FSE.
 	 *
+	 * @deprecated being replaced soon by an is_active static method - don't add new usages
 	 * @param string $theme_slug Theme slug to check support for.
 	 *
 	 * @return bool True if passed theme supports FSE, false otherwise.
 	 */
-	public function is_supported_theme( $theme_slug ) {
-		$theme_slug = $this->normalize_theme_slug( $theme_slug );
-		return in_array( $theme_slug, self::SUPPORTED_THEMES, true );
+	// phpcs:disable
+	public function is_supported_theme( $theme_slug = null ) {
```

Comment:
Nitpick, but is there a reason why we need to declare the function parameter?

Reply:
For backwards compatibility. Callers still expect this to be checking against a theme whitelist. We can remove the whole thing after we add in cleaner checks with `is_fse_active` and `is_fse_eligible` for a site.

Reply:
I mean, `$theme_slug` is never used in the function though.
Would PHP complain if someone called a function without parameters with a parameter?
E.g.
```php
function foo() {
  // ...
}

foo( 'bar' );
```

Reply:
You'd then question why `is_supported_theme()` has no args right? Let's just keep this as is for now, since we plan on removing the entire function